models/credmark/protocols/lending/compound/compound_v2.py

```diff
@@ -295,6 +300,11 @@ class CompoundV2GetPoolInfo(Model):
         }
     }
 
+    # APY calculation
+    ETH_MANTISSA = 1e18

```

Comment:
is this decimals?

Reply:
According to https://compound.finance/docs. It's an integer.
![image](https://user-images.githubusercontent.com/214410/167515258-ca59501a-886c-4953-9455-a0d83e5bc95b.png)